doc/manual/migrating.rst

```diff
@@ -47,8 +47,8 @@ Note that MNE has functions to read a variety of file formats, not just :func:`m
 Pitfalls
 --------
 
-* Python function often operate in-place. This means that the input to the function is modified.
+* Python methods often operate in-place. This means the object the method is called on is modified in-place (e.g., see the filter example above).
   This can be confusing to new users migrating from Matlab. However, it is also possible to ask MNE functions not to modify the input.
-  In this case, a copy of the input is made, which is operated upon and returned. Look out for the boolean argument ``copy`` in MNE functions.
+  To do this, call the ``copy`` method of the object (.e.g, use  :func:`raw_filtered = raw.copy().filter(l_freq, h_freq) <mne.io.Raw.filter>`).
```

Comment:
this should probably point to `<mne.io.Raw.copy>` instead since it's the point you're making